### PR TITLE
set tfars to same value for all points

### PR DESCRIPTION
**Pull request by dwastberg**
if a number is passed as argument dists of EmbreeScene.run then the tfars array is filled with that value.

### CODE REVIEW

File: pyembree/rtcore_scene.pyx [hunk 1]
```diff
@@ -50,6 +51,9 @@ cdef class EmbreeScene:
         if dists is None:
             tfars = np.empty(nv, 'float32')
             tfars.fill(1e37)
+        elif isinstance(dists,numbers.Number):
```
Comment (scopatz) [inline pyembree/rtcore_scene.pyx]: Please add a single space after the comma, a la PEP8

**Comment by scopatz** [comment]
Thanks for putting this in @dwastberg!

**Comment by scopatz** [comment]
Also can you please add a news entry for this and the other PR, as per the instructions at https://regro.github.io/rever-docs/news.html

**Comment by dwastberg** [comment]
Both things should be fixed now

**Pull request merged**